internal/osutils/autostart/autostart_mac.go

```diff
@@ -6,16 +6,27 @@ package autostart
 import (
 	"os"
 	"path/filepath"
+	"strings"
 
 	"github.com/ActiveState/cli/internal/assets"
 	"github.com/ActiveState/cli/internal/errs"
 	"github.com/ActiveState/cli/internal/fileutils"
+	"github.com/ActiveState/cli/internal/strutils"
 	"github.com/mitchellh/go-homedir"
 )
 
-const launchFileMacOS = "com.activestate.platform.state-tray.plist"
+const launchFileSource = "com.activestate.platform.state.plist.tpl"
 
-func (a *App) enable() error {
+var data = map[AppName]options{
+	Tray: {
+		launchFileName: "com.activestate.platform.state-tray.plist",
+	},
+	Service: {
+		launchFileName: "com.activestate.platform.state-svc.plist",
+	},
```

Comment:
This seems like it can be calculated from app.Name or app.Exec. That way autostart doesn't need any opinionated boilerplate.